xtask/src/main.rs

```diff
@@ -1,55 +1,91 @@
-use clap::CommandFactory;
+use clap::{CommandFactory, Parser};
 use clap_complete::{generate, Shell};
 use clap_mangen::Man;
 use snafu::{ResultExt, Snafu};
 use stackablectl::cli::Cli;
+use stackabled::api_doc::{ApiDoc, OpenApi};
 
-use std::{env, fs};
+use std::{
+    fs,
+    io::Write,
+    process::{Command, Stdio},
+};
+
+#[derive(clap::Parser)]
+#[allow(clippy::enum_variant_names)]
+enum XtaskCli {
```

Comment:
```suggestion
enum XtaskCommand {
```

This name fits better for what xtask actually does.

Reply:
Works for me, I was just trying to be consistent with it being called `Cli` everywhere else.

Reply:
43a044ae3e9b6f9a05ea3ec703ebf5c05de9c3e5